packages/control/state_machine.py

```diff
@@ -0,0 +1,23 @@
+from enum import IntEnum
+
+
+class StateMachine(IntEnum):
```

Comment:
Die Bezeichnung `StateMachine`ist zu allgemein und sagt nichts aus. Es sollte ein Bezug zu den möglichen Stati erkennbar sein. Der Zusatz "Machine" für eineinfaches Enum ohne weitere Funktionalität ist ebenfalls nicht glücklich gewählt.
```suggestion
class ChargePointState(IntEnum):
```

Reply:
Ist umbenannt